src/install_rules.ml

```diff
@@ -29,7 +29,8 @@ let gen_dune_package sctx ~version ~(pkg : Local_package.t) =
             |> List.map ~f:(fun lib ->
               let name = Lib.name lib in
               let dir_contents =
-                Dir_contents.get_without_rules sctx ~dir:(Lib.src_dir lib) in
```

Comment:
The only reason this is safe is because `lib` is always local. But this is something that reader's have to think about whenever they read the change. I wonder if it makes sense to introduce a `Lib.Local` submodule where we put operations that are only safe for local libraries.